src/main/kotlin/edu/carroll/polyominoes/service/account/RegisterService.kt

```diff
@@ -0,0 +1,31 @@
+package edu.carroll.polyominoes.service.account
+
+interface RegisterService {
+    /**
+     * Given a username, email, and rawPassword, creates a new account in the database.
+     *
+     * @param username: the valid username representing the user's account
+     * @param email: a valid email to be linked to the user's account
+     * @param rawPassword: a rawPassword for the user's account which will be hashed before storing
+     * @return true if an account was added to the database, false otherwise
+     */
+    fun createUser(username: String, email: String, rawPassword: String): Boolean
+
+    /**
+     * Given an email, determine if the user's given email already exist in the database.
```

Comment:
Spelling should be 'exists' (plural) in this context.  This english usage error **exists** in MANY comments throughout the codebase.